subprojects/core/src/integTest/groovy/org/gradle/api/internal/project/taskfactory/TaskPropertyNamingIntegrationTest.groovy

```diff
@@ -100,6 +100,7 @@ class TaskPropertyNamingIntegrationTest extends AbstractIntegrationSpec {
             }
         """
         when:
+        executer.expectDeprecationWarnings(2)
```

Comment:
Should we add a `@Nested Map` property to this test?

Reply:
The tests for nested properties are in `NestedInputIntegrationTest`, in particular the one for the map is [here](https://github.com/gradle/gradle/blob/9f9a664e4e2ae2539849bc10b4ce4e2e9dbcbbbc/subprojects/core/src/integTest/groovy/org/gradle/api/tasks/NestedInputIntegrationTest.groovy#L715).

I would currently leave them there.